src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin logoSize($size) {
```

Comment:
can I use this mixen for stars?) Looks like no

Reply:
@Andr1yK  But why?